library/spdm_requester_lib/libspdm_req_get_capabilities.c

```diff
@@ -210,30 +184,49 @@ static libspdm_return_t libspdm_try_get_capabilities(libspdm_context_t *spdm_con
     spdm_request->data_transfer_size = spdm_context->local_context.capability.data_transfer_size;
     spdm_request->max_spdm_msg_size = spdm_context->local_context.capability.max_spdm_msg_size;
```

Comment:
I think this fields should be filled according to spdm version

Reply:
Yes. The filed such as `CTExponet` will only exit after version 1.1. But I think it is better to submit a new issue to fix it. Thanks!

Reply:
Right. This task only *split* thing. Zero bug fix or logic change.

Reply:
I make a new issue for this. However, as @steven-bellock commented in https://github.com/DMTF/libspdm/issues/2412#issuecomment-1765441952, the request size is already writed appropriately, this have not side-effect. But, it can be improved as a guide